flocker/node/_deploy.py

```diff
@@ -397,6 +397,8 @@ def applications_from_units(result):
             running = []
             not_running = []
             manifestations = []  # Manifestation objects we're constructing
+            manifestation_paths = {dataset_id: path for (path, (dataset_id, _))
+                                   in available_manifestations.items()}
```

Comment:
Oh right but it's represented as a `FilePath` throughout and `FilePath` restricts us to bytes-only still.  Nevermind.